__test__/backup.test.ts

```diff
@@ -168,7 +168,7 @@ describe("Backup", function () {
     });
 
     it(`relative paths`, async () => {
-      const backupPath = "../";
+      const backupPath = "../foo";

```

Comment:
Previously, the `backupPath` was a parent directory of the test profile directory. As such, the relative path has been changed.